test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -430,11 +420,12 @@ export default class GutenbergEditorComponent extends AbstractEditorComponent {
 		return blockClass.Expect( this.driver, blockID );
 	}
 
+	// @todo This and the `getTitle` method in this class look too similar. Refactor to a single method.
 	async titleShown() {
 		const titleLocator = By.css( '.editor-post-title__input' );
 		await driverHelper.waitUntilElementLocatedAndVisible( this.driver, titleLocator );
 		const element = await this.driver.findElement( titleLocator );
-		return await element.getAttribute( 'value' );
+		return await element.getProperty( 'textContent' );
```

Comment:
Ditto, it also should make this backward-compatible.

Reply:
Backward compatibility is not that important here (unless we want to merge this before releasing Gutenberg, but there's no real tangible value in doing that), but it wouldn't hurt to have!